go.mod

```diff
@@ -88,26 +88,26 @@ replace (
 	github.com/openshift/api => github.com/openshift/api v0.0.0-20200323095748-e7041f8762a3
 	github.com/openshift/cluster-api => github.com/openshift/cluster-api v0.0.0-20191004085540-83f32d3e7070
 	github.com/securego/gosec => github.com/securego/gosec v0.0.0-20190709033609-4b59c948083c
-	k8s.io/api => k8s.io/api v0.18.0
-	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.18.0
-	k8s.io/apimachinery => k8s.io/apimachinery v0.18.0
-	k8s.io/apiserver => k8s.io/apiserver v0.18.0
-	k8s.io/cli-runtime => k8s.io/cli-runtime v0.18.0
-	k8s.io/client-go => k8s.io/client-go v0.18.0
-	k8s.io/cloud-provider => k8s.io/cloud-provider v0.18.0
-	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.18.0
-	k8s.io/code-generator => k8s.io/code-generator v0.18.0
-	k8s.io/component-base => k8s.io/component-base v0.18.0
-	k8s.io/cri-api => k8s.io/cri-api v0.18.0
-	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.18.0
-	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.18.0
-	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.18.0
-	k8s.io/kube-proxy => k8s.io/kube-proxy v0.18.0
-	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.18.0
-	k8s.io/kubectl => k8s.io/kubectl v0.18.0
-	k8s.io/kubelet => k8s.io/kubelet v0.18.0
+	k8s.io/api => k8s.io/api v0.18.6
+	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.18.6
+	k8s.io/apimachinery => k8s.io/apimachinery v0.18.6
+	k8s.io/apiserver => k8s.io/apiserver v0.18.6
+	k8s.io/cli-runtime => k8s.io/cli-runtime v0.18.6
+	k8s.io/client-go => k8s.io/client-go v0.18.6
+	k8s.io/cloud-provider => k8s.io/cloud-provider v0.18.6
+	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.18.6
+	k8s.io/code-generator => k8s.io/code-generator v0.18.6
+	k8s.io/component-base => k8s.io/component-base v0.18.6
+	k8s.io/cri-api => k8s.io/cri-api v0.18.6
+	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.18.6
+	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.18.6
+	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.18.6
+	k8s.io/kube-proxy => k8s.io/kube-proxy v0.18.6
+	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.18.6
+	k8s.io/kubectl => k8s.io/kubectl v0.18.6
+	k8s.io/kubelet => k8s.io/kubelet v0.18.6
 	k8s.io/kubernetes => k8s.io/kubernetes v1.18.0
```

Comment:
you are not updating this to 1.18.6 ?

Reply:
hah, these tiny little numbers ;)

Reply:
thanks for your patience @p0lyn0mial Done.